api/dp-openapi.yaml

```diff
@@ -690,6 +690,160 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /queues/{roundid}:
+    get:
+      tags:
+      - queues
+      description: Gets the release queues for a round
+      parameters:
+      - name: roundid
+        in: path
+        description: Round ID
+        required: true
+        schema:
+          type: string
+      - name: field
+        description: Queue field to return, if not set all fields are returned.
+          To request multiple fields, append with [] and pass one per desired
+          field (e.g. field[]=projectid&field[]=author).
+        in: query
+        schema:
+          type: string
+      - name: show
+        description: Which queues to show.
+          all - all queues
+          enabled - queues that are enabled
+          populated - queues that are enabled and have projects to release
+        in: query
+        schema:
+          type: string
+          enum: [all, enabled, populated]
+      - name: name
+        description: Release queue name substring to match. If not set, all queues are returned.
+          To request multiple possible queue names (results ORed together), append with
+          [] and pass one per desired name (e.g. name[]=Klingon&name[]=Shakespeare).
+        in: query
+        schema:
+          type: string
+      - name: group
+        description: Release queue group name substring to match. If not set all groups are
+          returned (as well as all matching queues in those groups). To match multiple possible
+          group names (results ORed together), append with [] and pass one per desired name
+          (e.g. group[]=PM&group[]=Math).
+        in: query
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Requested release queue list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/release_queue_detail'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /queue/{queueid}:
+    get:
+      tags:
+      - queues
+      description: Get release queue info, release stats, and projects
+      parameters:
+      - name: queueid
+        in: path
+        description: Queue ID
+        required: true
+        schema:
+          type: integer
+      - name: unheld_only
+        in: query
+        required: false
```

Comment:
`required: false` is the default so you don't need this string and we don't explicitly define it in most (but not all?) places in this file when its applicable.

Reply:
OK. I'll remove the ones I've introduced for now.